lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -376,6 +376,7 @@
     <datatype extension="star" type="galaxy.datatypes.images:Star" display_in_upload="true" />
     <datatype extension="peff" type="galaxy.datatypes.proteomics:PEFF" display_in_upload="true" />
     <datatype extension="toml" type="galaxy.datatypes.data:Text" subclass="true" display_in_upload="true"/>
+    <datatype extension="colab.tar" auto_compressed_types="gz" type="galaxy.datatypes.binary:CompressedArchive" subclass="true" display_in_upload="true"/>
```

Comment:
Why did you choose `auto_compressed_types`? 

Wondering if it would be worth to invest time in a sniffer (eg by content) or versioning. Do you think that a tar file of version x can be used by other versions.

Are those tar files an idea of colabfold or is this to pass sets of files from tool to tool?

Reply:
1. In case we wanted to update how we transfer between tools in later version
2. Possibly, and I can at least make one that looks for a specific file that should always be there between versions
3. This is entirely for use between the two tools since it has been divided from the original single tool to allow running the separate parts in different places. The second tool requires the full output directory from the firsrt